Doc/whatsnew/3.9.rst

```diff
@@ -147,6 +147,13 @@ Added constants :data:`~fcntl.F_OFD_GETLK`, :data:`~fcntl.F_OFD_SETLK`
 and :data:`~fcntl.F_OFD_SETLKW`.
 (Contributed by Dong-hee Na in :issue:`38602`.)
 
+imaplib
+-------
+
+The :class:`~imaplib.IMAP4` and :class:`~imaplib.IMAP4_SSL` now has
```

Comment:
This should be "... now have an optional *timeout* parameter for their constructors." Also, I'd probably drop the leading "The", or make it "The classes".